apischema/__init__.py

```diff
@@ -61,9 +61,10 @@
 
 try:
     from . import graphql  # noqa: F401
-
-    __all__.append("graphql")
-except ImportError:
+    
+    if not graphql.__version__.startswith("2"):
```

Comment:
There is no `graphql.__version__`, because `graphql` module imported here is `apischema.graphql` (not `graphql` of *graphql-core*)